make-based/app-sqlite/do.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+app_basename="sqlite"
+app_libs="musl sqlite"
+use_kvm="1"
+use_9p_rootfs="1"
+rootfs_9p="$PWD/sql_files"
```

Comment:
```suggestion
rootfs_9p="$PWD/files/sql_files"
```
I would create a new directory `files/sql_files` and add a dummy file so the user can test it. You can add the [`script.sql` script here](https://unikraft.org/docs/usage/make_build/#app-sqlite-for-x86_64).